src/components/Message/utils.tsx

```diff
@@ -183,56 +211,42 @@ export const areMessagePropsEqual = (props, nextProps) =>
   // Message wrapper layout changes
   nextProps.messageListRect === props.messageListRect;
 
-/**
- * @type {(nextProps: import('types').MessageComponentProps, props: import('types').MessageComponentProps ) => boolean} Typescript syntax
- */
-export const shouldMessageComponentUpdate = (props, nextProps) =>
+export const shouldMessageComponentUpdate = (
+  props: MessageComponentProps,
+  nextProps: MessageComponentProps,
+): boolean =>
   // Component should only update if:
   !areMessagePropsEqual(props, nextProps);
 
-/** @type {(message: import('stream-chat').MessageResponse | undefined) => boolean} */
-export const messageHasReactions = (message) =>
+export const messageHasReactions = (message?: MessageResponse) =>
```

Comment:
we're going to need generics in here, but no problem now, I'll add in when I run through all the Message files again